cmd/puppeth/ssh.go

```diff
@@ -85,7 +85,7 @@ func dial(server string, pubkey []byte) (*sshClient, error) {
 	}
 	auths = append(auths, ssh.PasswordCallback(func() (string, error) {
 		fmt.Printf("What's the login password for %s at %s? (won't be echoed)\n> ", login, server)
-		blob, err := terminal.ReadPassword(int(syscall.Stdin))
+		blob, err := terminal.ReadPassword(syscall.Stdin)
```

Comment:
`syscall.Stdin` is problematic because it's a different type on Linux and Windows. Lets use `int(os.Stdin.Fd())` here which will work on both platforms and won't cause `unconvert` to whine.

Reply:
Thanks, updated so.